api/general.service.ts

```diff
@@ -3107,6 +3107,48 @@ export class GeneralService {
         );
     }
 
+    /**
+     * 
+     * Immediately re-broadcast unconfirmed wallet transactions to all peers.
+     * @param observe set whether or not to return the data Observable as the body, response or events. defaults to returning the body.
+     * @param reportProgress flag to report request and response progress.
+     */
+    public resendwallettransactions(observe?: 'body', reportProgress?: boolean): Observable<Array<string>>;
```

Comment:
Is this method added on purpose? the PR title is `added aliasExists`, this one is a new method?